apps/ollama/0.5.1/docker-compose.yml

```diff
@@ -1,6 +1,6 @@
 services:
   ollama:
-    image: ollama/ollama:0.5.1
+    image: ollama/ollama:0.5.2
     container_name: ${CONTAINER_NAME}
     restart: unless-stopped
     ports:
```

Comment:
The provided patch shows an update to the `ollama` service's Docker image from version 0.5.1 to 0.5.2 in the `services` section of a configuration file. This change does not introduce any errors but offers a small performance improvement by upgrading to v0.5.2, which might contain bug fixes or optimizations. No major changes were detected.